tests/foreman/api/test_ansible.py

```diff
@@ -123,3 +129,56 @@ def test_positive_ansible_job_on_host(target_sat, module_org, rhel_contenthost):
     )
     result = target_sat.api.JobInvocation(id=job['id']).read()
     assert result.succeeded == 1
+
+    target_sat.api.Host(id=target_host.id).remove_ansible_role(data={'ansible_role_id': role_id})
+    host_roles = target_host.list_ansible_roles()
+    assert len(host_roles) == 0
```

Comment:
This test is highly vulnerable in the future to fail if any default roles are added to the satellite. Instead, you should verify that `role` in not in list of roles.

Reply:
we're already testing this things with default roles, and we need to sync/import roles to the satellite before assigning it to host, also we've handled roles removal in every ansible test, so it should list roles assigned in specific test only.